dts/bindings/charger/sbs,sbs-charger.yaml

```diff
@@ -0,0 +1,5 @@
+description: SBS 1.1 compliant charger (http://www.sbs-forum.org/specs)
+
+compatible: "sbs,sbs-charger"
```

Comment:
Should the compatible include the SBS version string?

Reply:
nack, I'm aiming to remain compatible with existing sbs,sbs-charger dts.